Makefile

```diff
@@ -128,8 +128,7 @@ clean-olena:
 
 # Checkout OCR-D/assets submodule to ./repo/assets
 repo/assets: assets-update
-	git submodule init
-	git submodule update
+	git submodule update --init repo/assets
```

Comment:
@kba, you could add that change to your pull request #24.

Reply:
Will do.